### PR TITLE
init rushstack zh translation

### PR DESCRIPTION
**Pull request by bvanjoi**

### CODE REVIEW

**Comment by bvanjoi** [comment]
@chengcyber PLAT, thanks.

File: websites/rushstack.io/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: Rush Stack 是什么
+hide_title: true
+---
+
+## Rush Stack 是什么
+
+**Rush Stack** 是：
+
+- **使命**：为 web 中大型 monorepo 仓库来提供可复用技术。
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md:10]:
```suggestion
- **使命**：为 web 项目的大规模 monorepo 仓库来提供可复用技术。

```

File: websites/rushstack.io/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- [tree-pattern](https://www.npmjs.com/package/@rushstack/tree-pattern): JavaScript 树结构的模式匹配，用于 lint 规则。
+- [一系列 webpack 插件](https://github.com/microsoft/rushstack/tree/master/webpack)，它们对构建页面应用很有用。
+
+## 与 Rush 什么关系？
+
+当你使用 [Rush](@rushjs/) 时， "Rush Stack" 内的组件是可选的。
+
+作为**构建调度器**，Rush 的主要工作是：
+
+- Establish a standard repo layout and familiar CLI, to facilitate developers who contribute to many different monorepos
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md:46]:
```suggestion
```

File: websites/rushstack.io/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+- 确保确定性和稳定的包安装（使用 Yarn，PNPM 或 NPM）
+- 按照正确的顺序并尽可能快的构建项目
+- 使用一系列策略来保证你的 monorepo 顺利地执行
+- 管理你的发布流程
+- 建立一个标准的仓库结构和熟悉的 CLI 工具，以便开发者可以在多个不同的 monorepo 中提交代码
+
+除了上述作用外，Rush 将其余的事情留给了你。个人项目中可以使用你喜欢的任何工具链来构建，这是非常灵活的。
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md:54]:
```suggestion
除了上述作用外，Rush 将其余的事情留给了你。每个项目可以分别使用你喜欢的任何工具链来构建，这是非常灵活的！
```

File: websites/rushstack.io/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+但是灵活性也有缺点，Node.js 工具因其令人纠结的技术栈而臭名昭著：选择编译工具、选择格式化工具、选择打包工具、选择包管理工具、选择任务引擎、选择测试工具、选择断言库等等。一旦你做出选择，就需要整合这些部件，而这些部件本身就是一个软件项目。随着规模的增大，这些成本也迅速增加。
+
+2019 年夏天，我们启动了 **Rush Stack**, 其目标是为上述问题提供一个可复用的解决方案。当然，你可以依然只使用 Rush 本身，但是当你厌倦于独自配置时，我们邀请你：
+
+- 舍去灵活性，来使用由工作在巨大的 monorepo 的开发者提供的方法
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md:60]:
```suggestion
- 权衡灵活性，选定一套能够工作在大型 monorepo 的方案
```

Comment (bvanjoi) [reply websites/rushstack.io/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md:60]: 原文是 

> Trade your flexibility for an opinionated approach backed by tooling experts who run huge monorepos

后半句我感觉翻译成 “选定一套由开发大型 monorepo 的专家提供的方案” 更好些。

**Comment by bvanjoi** [comment]
Thanks for @chengcyber , I found a lot of the original article forgot to delete, I'm so stupid😅

**Review by octogonz — APPROVED** [review]
I checked with Cheng and he's good with these changes

**Comment by octogonz** [comment]
Thank you @bvanjoi for doing this work!

**Pull request merged**